src/styles/blocks/stars.scss

```
@@ -0,0 +1,21 @@
+.rating-stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    width: 15px;
+    height: 15px;
+    margin-right: 4px;
+  
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-size: none;
+    background-position: center;
+  }
+}
+
+@for $gold-stars from 0 through $stars-count {
+  .rating-stars--active_#{$gold-stars} :nth-child(-n+#{$gold-stars}) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```

you have not 0 - star, and this shoud be in .rating-stars
```suggestion
@for $gold-stars from 1 through $stars-count {
  &--active_#{$gold-stars} :nth-child(-n+#{$gold-stars}) {
    background-image: url(../images/star-active.svg);
  }
}
```